gnmvidispine/vs_item.py

```diff
@@ -1111,8 +1116,8 @@ def commit(self):
         """
         path="{0}/metadata".format(self.parent.path())
         try:
-            self.request(path,method="PUT",body=self.as_xml(encoding="UTF-8"))
+            self.request(path,method="PUT",body=self.as_xml(encoding="utf8").decode("utf8"))
```

Comment:
so `self.request` expects a bytestring?

Reply:
Yes.

Reply:
I would verify that the argument is a bytestring (or str in py2) and raise a TypeError if not, so that the error is explicit

Reply:
I was wrong sorry. It looks like under Python 2 a unicode object is expected and under Python 3 a str object is expected. I have written in some code to raise a TypeError if the expected type is not found.